frontend/src/components/data-table/column-summary/column-summary.tsx

```diff
@@ -145,15 +145,20 @@ export const TableColumnSummary = <TData, TValue>({
         return null;
       case "time":
         return null;
-      case "string":
-        if (!spec) {
-          return (
-            <div className="flex flex-col whitespace-pre">
-              <span>unique: {prettyNumber(stats.unique, locale)}</span>
-            </div>
-          );
+      case "string": {
+        const lessThan5Categories =
+          typeof stats.unique === "number" && stats.unique <= 5;
+        // Number of categories will be easily visible from the chart, so don't show it here.
+        if (spec && lessThan5Categories) {
+          return null;
         }
-        return null;
+
+        return (
+          <div className="flex flex-col whitespace-pre -mt-1.5">
+            <span>categories: {prettyNumber(stats.unique, locale)}</span>
```

Comment:
maybe `unique` over `categories`? i don't always think of each item as a category